website/tsconfig.json

```diff
@@ -23,6 +23,7 @@
     "**/*.ts",
     "**/*.tsx",
     "components/**/*.tsx",
+    "components/*.tsx",
```

Comment:
@lazytype would have expected this to fix the typings in some components, and the typechecker points to this file as a reference, but still when I remove necessary types I'm not seeing expected errors

Reply:
`"**/*.ts"` and `"**/*.tsx"` should cover everything. it doesn't look like the tsconfig includes the rules it used to get from `common.tsconfig.json`, e.g.
```json
    "strictNullChecks": true,
    "noUnusedLocals": true,
    "strict": true,
```